libr/core/cmd_seek.c

```diff
@@ -134,7 +134,7 @@ static int cmd_seek(void *data, const char *input) {
 		if (input[1] && input[2]) {
 			if (core->io->debug) {
 				off = r_debug_reg_get (core->dbg, input + 2);
-				r_io_sundo_push (core->io, core->offset);
+				r_io_sundo_push (core->io, core->offset, core->print->cur);
```

Comment:
maybe add a getter for this value and ensure that the cur is enabled? otherwise it can result in trash values